CHANGELOG.md

```diff
@@ -7,122 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Add event tracking for Token Detection V2 ([#14441](https://github.com/MetaMask/metamask-extension/pull/14441))
 - Add search information for Theme dropdown ([#14476](https://github.com/MetaMask/metamask-extension/pull/14476))
-- Update version parsing to allow rollback release ([#14288](https://github.com/MetaMask/metamask-extension/pull/14288))
-- Update bug-report.yml ([#14466](https://github.com/MetaMask/metamask-extension/pull/14466))
-- Disable STX if a regular tx is in progress ([#14554](https://github.com/MetaMask/metamask-extension/pull/14554))
-- Fix development build scripts ([#14594](https://github.com/MetaMask/metamask-extension/pull/14594))
-- Update PULL_REQUEST_TEMPLATE.md ([#14547](https://github.com/MetaMask/metamask-extension/pull/14547))
-- Patch gridplus-sdk for perf fix ([#14582](https://github.com/MetaMask/metamask-extension/pull/14582))
-- Add applyLavaMoat build flag ([#14583](https://github.com/MetaMask/metamask-extension/pull/14583))
-- fix cross-fetch moderate vulnerability alert ([#14570](https://github.com/MetaMask/metamask-extension/pull/14570))
-- Fix browser.tab.get call ([#14550](https://github.com/MetaMask/metamask-extension/pull/14550))
-- Added getAccounts suppressUnauthorized param ([#14126](https://github.com/MetaMask/metamask-extension/pull/14126))
+
+### Changed
+- Update caret icon color on Home Page for darkmode, from black to white ([#14835](https://github.com/MetaMask/metamask-extension/pull/14835))
+- Code cleanup around tx logic: constants/transactions, `useTransactionDisplayData`, and `TransactionIcon` ([#14784](https://github.com/MetaMask/metamask-extension/pull/14784))
+- Removed draft transaction object from the send slide, for avoiding sync errors ([#14701](https://github.com/MetaMask/metamask-extension/pull/14701))
+- Update "secp256k1" for fast module init via lazy loading ([#14677](https://github.com/MetaMask/metamask-extension/pull/14677))
+- Update the color of the ledger alert text to adhere to design system guidelines ([#14674](https://github.com/MetaMask/metamask-extension/pull/14674))
+- Update version parsing to allow rollback release on Chrome ([#14288](https://github.com/MetaMask/metamask-extension/pull/14288))
+- Make existing component `SiteOrigin` reusable across all contexts where we surface a connected site's origin ([#14720](https://github.com/MetaMask/metamask-extension/pull/14720))
+- Swaps: handle an RPC provider delay, by returning an estimated post tx balance amount ([#14821](https://github.com/MetaMask/metamask-extension/pull/14821))
+- Swaps: reduce the default slippage from 3% to 2% ([#14863](https://github.com/MetaMask/metamask-extension/pull/14863))
+- Swaps: update STX status content page ([#14805](https://github.com/MetaMask/metamask-extension/pull/14805))
+  - Change STX status timer description dynamically based on the stxStatusDeadline param
```

Comment:
Suggestion to change "Change STX status timer description dynamically based on the stxStatusDeadline param" to "Make STX status timer behaviour dependent on API values"

Reply:
noted. Ty